GliaWidgets/View/Chat/Message/Content/Choice/ChoiceCardOptionStyle.swift

```diff
@@ -26,6 +28,7 @@ public final class ChoiceCardOptionStyle {
         normalBackgroundColor: UIColor,
         highlightedTextColor: UIColor,
         highlightedBackgroundColor: UIColor,
+        disabledTabsBorderColor: UIColor,
         disabledTextColor: UIColor,
         disabledBackgroundColor: UIColor
```

Comment:
I honestly don't remember why I decided to have this initializer instead of one that accepts three separate state styles - maybe due to common `textFont`. But now that I look at it, I feel like we should instead have an init that accepts `normal`, `highlighted` and `disabled`.